### DISCUSSION TITLE
v91.0.2

### DISCUSSION THREAD

**Discussion by kewisch** [kewisch/gdata-provider#276] [Announcements]
### Bug Fixes

* Accept if Calendars or Tasks are disabled in G Suite ([a2bfe48](https://github.com/kewisch/gdata-provider/commit/a2bfe48ce5086f54f415db1613c7ad056880f947)), closes [#65](https://github.com/kewisch/gdata-provider/issues/65)
* Avoid asking for changes when none have been made ([8226cd7](https://github.com/kewisch/gdata-provider/commit/8226cd7297cf3508080c10bf6af2d6862e868cf3)), closes [#11](https://github.com/kewisch/gdata-provider/issues/11)
* Correctly include organizer as attendee ([f05f6c9](https://github.com/kewisch/gdata-provider/commit/f05f6c91d43d64ced5334d3b4dd3a870e136b6ee)), closes [#62](https://github.com/kewisch/gdata-provider/issues/62)
* Correctly set cookie exceptions to make logins work ([ecc6781](https://github.com/kewisch/gdata-provider/commit/ecc678123430e32ff875e672d0bbda11a1a3e5eb)), closes [#43](https://github.com/kewisch/gdata-provider/issues/43)
* Don't show the migration dialog again when cancelled ([faf2124](https://github.com/kewisch/gdata-provider/commit/faf2124e6e4e4f058e8d163efa7ce635df9d0f7a))
* Make sure sync prefs are available at startup ([18acb63](https://github.com/kewisch/gdata-provider/commit/18acb63df11d613247f080536c4a4067969f7111))
* Port bug 1687612 - Fix new account OAuth window to be E10s capable. ([44a563a](https://github.com/kewisch/gdata-provider/commit/44a563a478e47f9c31e8c686998d445824a9b0a4))
* Properly display HTML from description ([6625aba](https://github.com/kewisch/gdata-provider/commit/6625aba20513a6f579261789ae8929e086328637))
* Refresh completed/hidden tasks as well ([3a0cf1a](https://github.com/kewisch/gdata-provider/commit/3a0cf1a7ff7dc71a745766e2941b9064f678e3ce)), closes [#72](https://github.com/kewisch/gdata-provider/issues/72)
* Remove error about aRequest.name not being implemented ([3e0162e](https://github.com/kewisch/gdata-provider/commit/3e0162e108db3bc227e6c2530133d41b19538f54))
* Resolve cyclic dependency and incorrect parameters in freebusy service calls ([d5c72bb](https://github.com/kewisch/gdata-provider/commit/d5c72bbe35f57f5e06a6c0defc273abfedb9810b)), closes [#147](https://github.com/kewisch/gdata-provider/issues/147)
* Still detect code if redirect is redirected to https ([c5e1090](https://github.com/kewisch/gdata-provider/commit/c5e109068ba9c92384729aa649098657a8f211fc)), closes [#227](https://github.com/kewisch/gdata-provider/issues/227)
* Trailing slash on completion URI ([dad61c9](https://github.com/kewisch/gdata-provider/commit/dad61c9b8ba813318d3bcae749cda8b2d71c9fc2))
* Use a different port number to avoid collisions ([5a57421](https://github.com/kewisch/gdata-provider/commit/5a57421e2b13f7dec136c6722b3902aa270c6c05)), closes [#240](https://github.com/kewisch/gdata-provider/issues/240)
* Use migrated preferences instead of purged legacy prefs ([8a86dd5](https://github.com/kewisch/gdata-provider/commit/8a86dd5cbd833a70813872995f4b2ed64681759e)), closes [#191](https://github.com/kewisch/gdata-provider/issues/191)
* Use oob mechanism for OAuth instead ([6b243be](https://github.com/kewisch/gdata-provider/commit/6b243be734707ab07758ca2057ebd5ad56cbe1f6)), closes [#240](https://github.com/kewisch/gdata-provider/issues/240)

### Features

* Go back to the session page if auth dialog is canceled ([4969f6f](https://github.com/kewisch/gdata-provider/commit/4969f6f1f4e0bfde97c83ab3ee9d2c12d8a01ef1))
* Invalidate session when password is removed ([abd16b1](https://github.com/kewisch/gdata-provider/commit/abd16b100636fec8b99dfac555f5317ff21e827d))
* Options UI for invitations options ([abc3a41](https://github.com/kewisch/gdata-provider/commit/abc3a417dcf7b6940f599140523f216eca846af4))
* Roundtrip color id so it is not removed ([90121ab](https://github.com/kewisch/gdata-provider/commit/90121abfbf3d25fd1119f9d665de4f6ec939c8d2)), closes [#236](https://github.com/kewisch/gdata-provider/issues/236)

<hr /><em>This discussion was created from the release <a href='https://github.com/kewisch/gdata-provider/releases/tag/v91.0.2'>v91.0.2</a>.</em>

**Comment by mtpmd** [comment]
I posted a few days ago in Issues that the provider in all 91 versions including 91.0.2 weren't accepting event reminder snoozes or dismissals, and got a reply that this issue was fixed in the main branch and couldn't be fixed in the legacy branch, and the issue closed.   However I can't find any information anywhere in FAQs or forums or discussion about what the reference point for "legacy" is,   

What exactly determines whether or not an event is handled by "legacy" code vs. "main branch" code, please? 

(Because for me, the problem is still happening after the official 91.0.2 release, so apparently if it really is fixed in main, all my events are being handled by the legacy branch, but I can't figure out what I would need to do route them through the main branch instead [even removing the calendar and bringing in everything as freshly imported into a naive calendar, so not I would have thought any "legacy" events from anything, doesn't fix it])?

**Reply by kewisch** [reply]
All release versions right now are on the "legacy branch". There is no way to enable this at runtime. The "main branch" is a completely rewritten extension that has not been released yet. Unfortunately the reminder issue will not be fixed until I find time to complete the work from the rewrite.

**Comment by bpinsky123** [comment]
I've been trying to add my Google calendar with the latest version of the provider.  I have TB 91.2.0.  It goes through the whole authentication process, asks for my MFA, and presents me the dialog with the checkboxes for permission to access my calendar and tasks.  After that, I simply get an empty dialog box instead of a list of calendars to choose to subscribe. Error console and empty dialog attached here
![Clipboard01](https://user-images.githubusercontent.com/43283231/138001156-db6de477-1390-4754-ba65-543b816c276e.jpg)
![Clipboard02](https://user-images.githubusercontent.com/43283231/138001169-3e073a0a-355e-41de-a893-8d8930de0464.jpg)

.
